### PR TITLE
Add page for constants/immutables

### PR DESCRIPTION
**Pull request by EndymionJkb**
This also updates some formerly "pending" sections, given that we've launched (there may be more like this!), and adds some disclaimers arising from issues found during the Cantina competition, especially relating to buffers and the utility of factory contracts for pools.

### CODE REVIEW

File: docs/build/build-an-amm/deploy-custom-amm-using-factory.md [hunk 1]
```diff
@@ -7,7 +7,11 @@ title: Deploy a Custom AMM Using a Factory
...
+
+Although core Balancer pool factories do not do this, as they are meant to be generic and support many different use cases, including separate funding, consider also initializing through the factory (or at least in the same transaction as the create). Factories could even inspect the on-chain conditions during deployment, and revert if they are unfavorable (e.g., nested pool rates have been manipulated), further reducing the risks of front-running or other types of interference. Regarding initialization, it is also recommended to initialize ERC4626 buffers *before* deploying any pools with corresponding wrapped tokens. Initialization is the step that sets the proportion of underlying and wrapped tokens; thereafter, liquidity can only be added proportionally. To ensure the pools work as intended, buffers should be created and funded by sponsors prior to deploying pools designed to use them.
```
Comment (mkflow27) [inline docs/build/build-an-amm/deploy-custom-amm-using-factory.md:12]: Great additions, thanks.

File: docs/concepts/vault/buffer.md [hunk 2]
```diff
@@ -17,24 +17,63 @@ A significant benefit of the Vault's liquidity buffers is that Liquidity Provide
...
+
+The general idea is that funding parties would be DAO partners (starting with the Balancer DAO) who are motivated to support the platform and promote use of their protocol or yield-bearing token. Immediate access to Balancer liquidity and gas-efficient trades for community members should be enough to entice DAOs and lending protocols to contribute. They would be potentially sacrificing yield on a relatively small position, in hopes that the UX improvement and gas savings for retail users will attract disproportionately greater trade volume and TVL.
+
+Note also that since buffers are not pools, buffer liquidity is not tokenized. Instead of receiving BPT in return for a buffer deposit, the system calculates buffer "shares," and maintains internal accounting of the total shares issued, and the share balance per LP. An important consequence of this is that these shares cannot be transferred. Only the account (or contract) that added liquidity will be able to remove it, so if a contract is involved in providing liquidity, care must be taken to ensure that contract supports all necessary operations.
+
```
Comment (mkflow27) [inline docs/concepts/vault/buffer.md:35]: Thanks for these additions. Really provide more context.

**Review by mkflow27 — APPROVED** [review]
lgtm

File: docs/developer-reference/contracts/vault-config.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+v3 uses the same Weighted Math as v2, and the same minimum weight, so the same limits apply. Essentially, the total invariant cannot increase more than 3x, or decrease to less than 0.7x of the current value in a single operation.
+
+# Stable Pool Constants
+
+## Minimum/Maximum Amplification Parameter
+
+`MIN_AMP` is 1; `MAX_AMP` is 5000, same as v2. These values ultimately arise from the math in Curve's StableSwap. Higher values "flatten" the price curve (i.e., have a greater range were the tokens trade at essentially 1:1), and lower values make it more sensitive to the balances, and behave more like the Weighted Math price curve. Higher liquidity and lower volatility pools can generally have higher Amplification Parameters.
+
+There is also an `AMP_PRECISION` constant, set to 1000. The integer 1-5000 values are multiplied by this factor for greater precision in calculation; the Amplification Parameter is used for the invariant computation.
```
Comment (MattPereira) [inline docs/developer-reference/contracts/vault-config.md:146]: Perfect layman's explanation for pool creation UI links! :pray:

**Pull request merged**